src/components/Search/SearchAutocomplete.tsx

```diff
@@ -12,27 +13,26 @@ function getOptionSelected(option: Region, selectedOption: Region) {
 const SearchAutocomplete: React.FC<{
   locations: Region[];
   filterLimit: number;
-}> = ({ locations, filterLimit }) => {
+  setHideMapToggle?: any;
+}> = ({ locations, filterLimit, setHideMapToggle }) => {
   const [input, setInput] = useState('');
-  const [isZip, setIsZip] = useState(false);
+  /* We only check for a zipcode match when the input is all numbers and has a length of 5: */
+  const [checkForZipcodeMatch, setCheckForZipcodeMatch] = useState(false);
   const [noOptionsCopy, setNoOptionsCopy] = useState('No location found');
 
   const onInputChange = (e: any, value: string) => {
     setInput(value);
     const isStringOfDigits = /^\d+$/.test(value);
     if (isStringOfDigits) {
       setNoOptionsCopy('Enter a valid 5-digit zip code');
-      if (value.length === 5) setIsZip(true);
-      else setIsZip(false);
+      if (value.length === 5) setCheckForZipcodeMatch(true);
+      else setCheckForZipcodeMatch(false);
     } else setNoOptionsCopy('No location found');
   };
 
   const stringifyOption = (option: Region) => {
-    if (isZip) {
-      if ((option as County).zipCodes) {
-        return `${(option as County).zipCodes.join(' ')}`;
-      }
-      return option.name;
+    if (checkForZipcodeMatch && (option as County).zipCodes) {
```

Comment:
does this function the same as `(option instanceof County)`? I'm not sure I know the as syntax well enough to know how it behaves if it's not a county.